man/systemctl.xml

```diff
@@ -663,6 +663,9 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           <option>ExecReload=</option>, <option>ExecStartPre=</option>, etc.) run in distinct namespaces.
           </para>
 
+          <para>If the kernel supports it, the mount will be applied beneath the mount point, which will be
+          lazy umounted afterwards, atomically replacing the previous mount, if any.</para>
```

Comment:
this sounds like an implementation detail to me, i'd not mention this.

Reply:
We should mention it will replace the existing mount though right? How would you like this to be worded?

Reply:
just say "If supported by the kernel any prior mount on the selected target will be replaced by the new mount. If not supported any prior mount will be over-mounted, but remain pinned, however inaccessible". or so